### DISCUSSION TITLE
i'm trying to automate ios but i can't get it to work!

### DISCUSSION THREAD

**Discussion by lex0tan** [YueChen-C/py-ios-device#125] [Q&A]
Starting from the fact that i don't know what to put into:

wda_bundle_id
device_id (this one is imei i guess?)
app

i got the code from https://github.com/YueChen-C/py-ios-device/blob/main/doc/%E4%BD%BF%E7%94%A8%E6%96%87%E6%A1%A3.md#ios-%E8%87%AA%E5%8A%A8%E5%8C%96

i'm on windows, also, is there a way to simulate clicks and change the gps location? thank you 👍

**Comment by YueChen-C** [comment]
try pyidevice simulate-location 

device_id == udid

**Reply by lex0tan** [reply]
pyidevice isn't working for me:

Err: pyidevice is not recognized as an internal or external command

**Reply by YueChen-C** [reply]
Please read README.md carefully

pip install pyidevice

**Reply by lex0tan** [reply]
ok apparently i fixed it, now i need to know what i should put into

wda_bundle_id (i need xcode to get this?)
app (something like com.cardify.tinder? but for which app?)

**Comment by lex0tan** [comment]
@YueChen-C any help with finding  wda_bundle_id?

**Reply by YueChen-C** [reply]
@lex0tan `wda` usually refers to https://github.com/appium/WebDriverAgent 
You need to compile and specify a bundleid yourself，
